assets/app/views/create/nextSteps.html

```diff
@@ -0,0 +1,102 @@
+            </div>  
+          </div>
+        </div>
+
+        <div class="buttons gutter-top" ng-if="!task.hasErrors">
+          <a class="btn btn-primary btn-lg" href="project/{{projectName}}/overview">Go to overview</a>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="row" ng-controller="TasksController">
+      <div ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length" class="col-md-12">
+        <h2>Things you can do</h2>
+        <p>Go to the <a href="project/{{projectName}}/overview">overview page</a> to see more details about this project. Make sure you don't already have <a href="project/{{projectName}}/browse/services">services</a>, <a href="project/{{projectName}}/browse/builds">build configs</a>, <a href="project/{{projectName}}/browse/deployments">deployment configs</a>, or other resources with the same names you are trying to create. Refer to the <a target="_blank" href="{{'new_app' | helpLink}}">documentation for creating new applications</a> for more information.</p>
+
+        <h3>Command line tools</h3>
+        <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
+
+        <pre class="code prettyprint">oc login {{baseUrl}}
+oc project {{projectName}}
+oc build-logs -h
+oc logs -h</pre>
+
+        <p>For more information about the command line tools, check the <a target="_blank" href="{{'cli' | helpLink}}">CLI Reference</a> and <a target="_blank" href="{{'basic_cli_operations' | helpLink}}">Basic CLI Operations</a>.</p>
+      </div>
+
+      <div ng-if="allTasksSuccessful(tasks())" ng-class="createdBuildConfigWithGitHubTrigger() ? 'col-md-6' : 'col-md-12'">
+        <h2>Manage your app</h2>
+        <p>The web console is convenient, but if you need deeper control you may want to try our command line tools.</p>
+        
+        <h3>Command line tools</h3>
+        <p><a target="_blank" href="{{'get_started_cli' | helpLink}}">Download and install</a> the <code>oc</code> command line tool. After that, you can start by logging in, switching to this particular project, and displaying an overview of it, by doing:</p>
+
+        <pre class="code prettyprint">oc login {{baseUrl}}
+oc project {{projectName}}
+oc status</pre>
+
+        <p>For more information about the command line tools, check the <a target="_blank" href="{{'cli' | helpLink}}">CLI Reference</a> and <a target="_blank" href="{{'basic_cli_operations' | helpLink}}">Basic CLI Operations</a>.</p>
+      </div>
+
+      <div ng-if="createdBuildConfigWithGitHubTrigger()" class="col-md-6">
+        <h2 ng-if="createdBuildConfig">Making code changes</h2>
+        <div ng-repeat="trigger in createdBuildConfig.spec.triggers">
+          <div ng-switch="trigger.type">
```

Comment:
you can actually even move the ng-if onto the same div as the ng-repeat, and then it will only generate a div for the one trigger that is actually the github webhook